registry/corestake/calldata-corestake.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://eips.ethereum.org/assets/eip-7730/erc7730-v1.schema.json",
+  "context": {
+    "contract": {
+      "abi": "https://openapi.coredao.org/api/contracts/abi_of_verified_contract/0xf5fA1728bABc3f8D2a617397faC2696c958C3409",
+      "deployments": [
+        {
+          "chainId": 1116,
+          "address": "0xf5fA1728bABc3f8D2a617397faC2696c958C3409"
+        }
+      ]
+    }
+  },
+  "metadata": {
+    "name": "CoreStake",
+    "version": "1.0.0",
+    "description": "Staking contract for CORE tokens"
+  },
+  "display": {
+    "formats": {
+      "mint(address)": {
+        "description": "Stake CORE tokens to a validator",
+        "fields": [
+          {
+            "path": "validator",
+            "label": "Validator Address",
+            "format": "addressName"
+          }
+        ]
+      },
+      "redeem(uint256)": {
+        "description": "Unstake CORE by redeeming staked tokens (stCORE)",
+        "fields": [
+          {
+            "path": "stCore",
+            "label": "Amount to Redeem (in stCORE)",
+            "format": "tokenAmount",
+            "params": {
+              "token": "0xf5fA1728bABc3f8D2a617397faC2696c958C3409"
```

Comment:
This needs to be the address of the ERC20 contract. It doesn't seem the contract is implementing IERC20.

https://scan.coredao.org/address/0x62c5e03a5bfa0d6af08b81165a9eb87d1c8b8a0b#code

I guess the address is supposed to be 0x2374959159a5520fbb778ec192a26dee277d2df0